api/axis.py

```diff
@@ -336,7 +336,7 @@ def __init__(self, _db, collection, api_class=Axis360API):
         else:
             self.api = api_class(_db, collection)
 
-    def process_batch(self, identifiers):
+    def process_items(self, identifiers):
```

Comment:
This is an API change. process_batch still exists but it takes an offset now. We want to use the default process_batch which finds the actual identifiers and calls process _items.